backend/open_webui/retrieval/models/base_reranker.py

```
@@ -0,0 +1,8 @@
+from abc import ABC, abstractmethod
+from typing import Optional, List, Tuple
```

Ruff (0.11.9)

`typing.List` is deprecated, use `list` instead

(UP035)